App.js

```diff
@@ -23,21 +25,51 @@ const client = new ApolloClient({
 });
 
 export default function App() {
+  const [hasToken, retrieveToken] = useState(false);
+  const [userToken, getUserToken] = useState([]);
+
+  function getStorage(){
+    //localStorage.removeItem('jwtToken');     //This is how u delete the token
+    localStorage.getAllFromLocalStorage().then(() => {
+      if(localStorage.getItem('jwtToken')){
+        const token = localStorage.getItem('jwtToken');
+        getUserToken(jwt_decode(token));
+        retrieveToken(true);
+      }
+    })
+    .catch(err => {
+      console.warn(err);
+    })
+  }  
+  
+  if(!hasToken){
+    getStorage();
+    //console.log(userToken)
```

Comment:
Comments are fine but don't leave commented-out code like this (except line 32, leave that for now)